core/drivers/amd/mailbox_driver.c

```diff
@@ -57,15 +58,17 @@ TEE_Result mailbox_open(uint32_t remote_id, size_t payload_size)
 	if (!ipi.req) {
 		EMSG("Failed to map request buffer");
 		ipi.remote = 0;
-		return res;
+		return TEE_ERROR_GENERIC;
 	}
 
 	ipi.rsp = core_mmu_add_mapping(MEM_AREA_RAM_SEC,
 				       rsp_addr,
 				       payload_size);
 	if (!ipi.rsp) {
 		EMSG("Failed to map response buffer");
-		/* Remove mapping for request buffer */
+		/* Remove mapping for request buffer.
+		 * Not checking return value res, since already in error case.
+		 */
```

Comment:
```patch
-		/* Remove mapping for request buffer.
+		/*
+		 * Remove mapping for request buffer.
 		 * Not checking return value res, since already in error case.
 		 */